mentoring/how_to_give_great_feedback.md

```diff
@@ -9,32 +9,32 @@ If you haven't, work your way through [Choosing a solution to mentor](/docs/ment
 
 You're about to give your first feedback to a student.
 This can be exciting but also pretty daunting.
-A lot of (the best!) mentors suffer from [Impostor Syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) at this stage, and give up before they start.
-If you feel like that, it's totally understandable, but also almost always unnecessary.
-By the fact you're worrying about whether your feedback is going to be good enough, it's likely that you'll give thoughtful humble comments that students really appreciate.
-And in the worst case, a student can always end the discussion, and wait for another mentor, so let's give it a go!
+A lot of (the best!) mentors suffer from [Impostor Syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome) at this stage and give up before they start.
+If you feel like that, it's totally understandable but also almost always unnecessary.
+By the fact that you're worrying about whether your feedback is going to be good enough, it's likely that you'll give thoughtful and humble comments that students will really appreciate.
+And in the worst case, a student can always end the discussion and wait for another mentor. So let's give it a go!
 
 ## Mindset
 
 The first thing to keep in mind is having the right mindset as a mentor.
-Students are hoping you can help improve their knowledge, and tell them things that they don't know.
+Students are hoping you can help improve their knowledge and tell them things that they don't know.
 Your job isn't to "mark" their work, it's to use their solution as a basis for unlocking ideas that they are not familiar with or are struggling with.
 
 It's also worth remembering that most Exercism users are experienced developers learning a new language.
-So although their code might look bad, that may well be because they don't know how to write this language.
+So although their code might look bad to you, that may well be because they simply aren't experienced writing _this_ language.
 Similar code in a different language might be totally idiomatic and correct.
-Conversations tend to go much better when they are approached as two peers discussing a topic, rather than one person showing off their knowledge to another.
+Conversations tend to be more productive when they are approached as two peers discussing a topic rather than one person showing off their knowledge to another.
```

Comment:
I like the comma that was here before.  The sentence could have ended where that comma was, and so the pause is to extend on the idea that was presented up to that point.